app/V1Module/router/RouterFactory.php

```diff
@@ -377,6 +377,16 @@ private static function createAssignmentSolutionsRoutes(string $prefix): RouteLi
             "AssignmentSolutions:evaluationScoreConfig"
         );
 
+        $router[] = new GetRoute("$prefix/<id>/review", "AssignmentSolutionReviews:");
+        $router[] = new PostRoute("$prefix/<id>/review", "AssignmentSolutionReviews:update");
+        $router[] = new DeleteRoute("$prefix/<id>/review", "AssignmentSolutionReviews:remove");
+        $router[] = new PostRoute("$prefix/<id>/review-comment", "AssignmentSolutionReviews:newComment");
+        $router[] = new PostRoute("$prefix/<id>/review-comment/<commentId>", "AssignmentSolutionReviews:editComment");
```

Comment:
PUT?

Reply:
I thought about it and I think arguments can be made both for POST and PUT (I assume we are talking about the new comment, not the update). If you have a good point about why this should be PUT and should not be POST, I am willing to change it :)

Reply:
I am looking at other endpoints and we are not using PUT anywhere (with 2 exceptions)... so it does not make sense to make this one PUT if we generally do not use it...

But I am talking about updating the comment, updates are usually done in REST API with PUT, POST is for creation...

Reply:
True, but
- PUT's semantics is often perceived as "replace", which is not entirely true for this endpoint (although this is only a weak argument)
- unlike POST, PUT has no input limit in the PHP input wrapper, thus we use it only for file uploads
So I would prefer consistency over ideology in this case.

Reply:
Yes, as I said, I would too...